src/test/cls_rbd/test_cls_rbd.cc

```diff
@@ -84,10 +85,35 @@ class TestClsRbd : public ::testing::Test {
 
 };
 
+class TestClsRbdCg : public ::testing::Test {
```

Comment:
Creating pools is very expensive, time-wise, when not using the unit test harness.  I can't think of a reason why you need unique pools.